pandas/tests/groupby/test_grouping.py

```diff
@@ -628,6 +628,49 @@ def test_groupby_empty(self):
         # check name
         assert s.groupby(s).grouper.names == ["name"]
 
+    def test_groupby_level_index_value_all_na(self):
+        # issue 20519
+        df = pd.DataFrame([["x", np.nan, 1]], columns=["A", "B", "C"]).set_index(
+            ["A", "B"]
+        )
+        result = df.groupby(level=["A", "B"]).sum()
+        expected = DataFrame(
+            data=[],
+            index=MultiIndex(
+                levels=[["x"], []],
+                codes=[[], []],
+                names=["A", "B"]
+            ),
+            columns=["C"]
+        )
+        tm.assert_frame_equal(
+            result,
+            expected,
+            check_index_type=False,
+            check_dtype=False
+        )
+
+        df = pd.DataFrame(
```

Comment:
Whats the difference between this and the previous test? Multiple all-missing levels?

Reply:
What i want is to show difference all the values in a level of a MultiIndex is missing and some values in a level of a MultiIndex is missing. Some values which are missing in a level of a MultiIndex could be handled before. But all the values in a level of a MultiIndex couldn't.

Reply:
Is that not already covered in another test? Good to have but I agree it's also not very clear why this is part of the test at the outset